test-site/sample.env

```diff
@@ -0,0 +1 @@
+REACT_APP_LIVE_API_KEY=[[Your Live API Key]]
```

Comment:
nit: can you also add a similar entry for `REACT_APP_MAPBOX_API_KEY`? looks like we forgot to add it previously for the test site